pebble.go

```diff
@@ -32,6 +29,10 @@ func NewPebbleDB(name string, dir string) (*PebbleDB, error) {
 }
 
 func NewPebbleDBWithOpts(name string, dir string, opts *pebble.Options) (*PebbleDB, error) {
+	// configure pebble
+	opts.MemTableSize = 256 << 20           // 256 MB
```

Comment:
this should be a separate PR. Ideally 3 PRs: 1) updating Go 2) Removing dbs other than goleveldb / pebbledb 3) configuring default options